src/core-plugins/CoreCommands.ts

```diff
@@ -32,8 +32,8 @@ export default class CoreCommands extends Plugin {
             description: "Lists all installed Aliucord plugins",
             options: [],
             execute: (args, ctx) => {
-                const enabledplugins = Object.values(plugins).filter(p => p.enabled === true).map(p => p.manifest.name);
-                const disabledplugins = Object.values(plugins).filter(p => p.enabled === false).map(p => p.manifest.name);
+                const enabledplugins = Object.values(plugins).filter(p => p.enabled).map(p => p.manifest.name);
+                const disabledplugins = Object.values(plugins).filter(p => !p.enabled).map(p => p.manifest.name);
```

Comment:
can you cache the intermediate Object.values()
also camelCase names
didnt notice this before